etc/make/local.mk

```diff
@@ -2,9 +2,11 @@
 # lint results.
 LINT_EXCLUDES_EXTRAS =
 
-# Regex for 'go vet' rules to ignore
-GOVET_IGNORE_RULES = \
-	possible formatting directive in Error call
+# Regexes for 'go vet' rules to ignore
+FILTER_GOVET := grep -v \
+	-e '^\#' \
+	-e 'possible formatting directive in Error call' \
+	-e 'refers to unknown identifier'
```

Comment:
Should we narrow this to `Example.+refers to unknown identifier`? I'm not sure what the complete list of vet errors is, but I can imagine a whole host of legit errors that include this phrase.

Reply:
Done